auth/build.gradle

```diff
@@ -1,19 +1,20 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlinx-serialization'
-apply plugin: 'org.jetbrains.dokka'
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'kotlinx-serialization'
+    id 'org.jetbrains.dokka'
+}
 
 android {
     namespace 'com.alfresco.auth'
     defaultConfig {
-        versionName "0.8.1"
+        versionName "0.8.3"
```

Comment:
The version should be 0.8.2.

Reply:
0.8.2 already released for auth0 changes